todolist/src/component/registertodo/TodoSave.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { forwardRef } from 'react';
+import { useSelector } from 'react-redux';
+
+const TodoSave = forwardRef(({ onClick }, refFn) => {
+  return (
+    <div>
+      <input type='text' ref={refFn} />
+      <button type='button' onClick={onClick}>
+        저장하기
+      </button>
+    </div>
+  );
+});
```

Comment:
사소하지만, 보통 네이밍을 refFn보다, ref를 사용합니다.
```suggestion
const TodoSave = forwardRef(({ onClick }, ref) => {
  return (
    <div>
      <input type='text' ref={ref} />
      <button type='button' onClick={onClick}>
        저장하기
      </button>
    </div>
  );
});
```